### ISSUE TITLE
Wind turbine visualised in WebPort for Ihus

### ISSUE THREAD

**Issue by Vivesoll** [Power2U/ICT_Task_Force#204]

**Comment by Vivesoll** [comment]
➤ Kristofer Sjöholm commented:

The integration need to be done first.

**Comment by Vivesoll** [comment]
➤ Kristofer Sjöholm commented:

On hold. Will be setup when the meters have been settled.

**Comment by Vivesoll** [comment]
➤ Arshad Saleem commented:

waiting for meter address [erik.stigenius](https://app.asana.com/0/570329100390061/list)

**Comment by Vivesoll** [comment]
➤ Arshad Saleem commented:

combine it with Flexview. [eric.jensen](https://app.asana.com/0/193610869715847/list)
